### PR TITLE
fix: Add tracking to the UserInformation component

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR updates the `decentraland-gatsby` dependency to add tracking to it.

### CODE REVIEW

File: src/entities/Place/model.ts [hunk 1]
```diff
@@ -354,8 +354,8 @@ export default class PlaceModel extends Model<PlaceAttributes> {
       hotScenePlaces.reverse()
     }
 
-    const from = numeric(offset || 0, { min: 0 })
-    const to = numeric(from + (limit || 100), { min: 0, max: 100 })
+    const from = numeric(offset || 0, { min: 0 }) ?? 0
+    const to = numeric(from + (limit || 100), { min: 0, max: 100 }) ?? 100
```
Comment (braianj) [inline src/entities/Place/model.ts:358]: do we need this? we are already sending 0 or 100 to the function

File: src/entities/Place/routes/getPlaceList.ts [hunk 2]
```diff
@@ -61,8 +61,8 @@ export const getPlaceList = Router.memo(
 
     const options: FindWithAggregatesOptions = {
       user: userAuth?.address,
-      offset: numeric(query.offset, { min: 0 }),
-      limit: numeric(query.limit, { min: 0, max: 100 }),
+      offset: numeric(query.offset, { min: 0 }) ?? 0,
+      limit: numeric(query.limit, { min: 0, max: 100 }) ?? 100,
```
Comment (braianj) [inline src/entities/Place/routes/getPlaceList.ts:65]: same as before

**Review by braianj — APPROVED** [review]
👍

File: src/entities/Place/model.ts (hunk 1 above)
Comment (LautaroPetaccio) [reply src/entities/Place/model.ts:358]: The function returns `number | null`, making us provide a default number.

File: src/entities/Place/routes/getPlaceList.ts (hunk 2 above)
Comment (LautaroPetaccio) [reply src/entities/Place/routes/getPlaceList.ts:65]: Same as above.

**Pull request merged**